README.md

```
@@ -34,114 +34,94 @@
 
 ## 🛠 The 3 Tools
 
+**Simply talk to your AI assistant in plain English.** No complex commands or JSON needed - just describe what you want to do with your Docker infrastructure.
+
 ### Tool 1: `docker_hosts`
-Manage your Docker hosts and connectivity.
+Simplified Docker hosts management tool.
 
 **Actions:**
 - `list` - List all configured Docker hosts
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)